### ISSUE TITLE
fails to build w/ gcc11

### ISSUE THREAD

**Issue by philipbjorge** [boborbt/priority_queue_cxx#5]
[gcc11 made c++17 the default](https://www.phoronix.com/scan.php?page=news_item&px=GCC-11-Cpp-17-Default#:~:text=But%20with%20GCC's%20C%2B%2B,17%20as%20the%20default%20dialect.) and this extension fails to build

```
current directory: /home/philip/.asdf/installs/ruby/2.7.6/lib/ruby/gems/2.7.0/gems/priority_queue_cxx-0.3.4/ext/fast_containers
make "DESTDIR="
compiling FastContainers.cpp
In file included from FastContainers.cpp:22:
fc_pq.h:78:51: error: ISO C++17 does not allow dynamic exception specifications
   78 |   PQueueIterator iterator_next(PQueueIterator it) throw(PQueueException);
      |                                                   ^~~~~
make: *** [Makefile:237: FastContainers.o] Error 1

make failed, exit code 2
```

Workaround:
`gem install priority_queue_cxx -- -v "0.3.4" --with-cxxflags="-std=c++14"`

**Comment by boborbt** [comment]
Hi @philipbjorge, thanks for the heads-up. I fixed the problem and published the gem with version 0.3.5. 

I am working on a Mac where the default compiler (clang++) does not report the problem, so I had to reproduce it with gcc. On my site everything seems ok, it would be great if you could check the new version and confirm that everything is ok also on your site.

Thanks in advance.

I am going to close this issue, feel free to reopen it if you still encounter problems.

**Comment by philipbjorge** [comment]
@boborbt -- This worked perfectly! Thank you for the update!
